src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -24,8 +25,23 @@ internal class ProcessMetrics
     internal static readonly AssemblyName AssemblyName = typeof(ProcessMetrics).Assembly.GetName();
     internal static readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
 
+    private const string MetricPrefix = "process.runtime.dotnet.";
```

Comment:
```suggestion
    private const string MetricPrefix = "process.";
```

Reply:
I saw from this [doc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#attributes) indicating that  resource attributes SHOULD be included on runtime metric events as appropriate. And java uses this convention, for example: process.runtime.jvm.memory.usage.
Which one should we follow?

Reply:
This is about process instrumentation instead of runtime instrumentation? no?